app/Entities/FavoriteList/FavoriteListingPresenter.php

```diff
@@ -71,6 +71,10 @@ private function listingClosing()
 	*/
 	private function filterMarkup()
 	{
+		global $post;
+        	$post = get_post( $this->favorite, OBJECT );
+        	setup_postdata( $post );
+		
```

Comment:
I think its important that we reset the global post data to what it was previously after hijacking the `$post` variable as this could have adverse side effects.   I'd imagine using `wp_reset_postdata` would do the trick. 

[Codex](https://codex.wordpress.org/Function_Reference/wp_reset_postdata)